.gitignore

```diff
@@ -7,3 +7,4 @@ local-browsers
 .synthetics
 react-app-env.d.ts
 eslint-junit.xml
+demo-app/package-lock.json
```

Comment:
It should be part of .npmrc file that ignores lock files.

Reply:
I'll add it but it seems only to work when running an `npm install` from the `demo-app` directory, so we're going to want to keep ignoring it if we aren't going to commit it.